composer.json

```diff
@@ -45,20 +45,23 @@
         "laminas/laminas-zendframework-bridge": "^1.0"
     },
     "require-dev": {
-        "doctrine/doctrine-mongo-odm-module": "^0.11 || ^1.0",
-        "doctrine/doctrine-orm-module": "^1.1 || ^2.1.3",
+        "doctrine/dbal": "^2.12.1 || ^3.0.0",
+        "doctrine/orm": "^2.7.5",
+        "doctrine/doctrine-mongo-odm-module": "^1.0 || ^2.0.2 || ^3.0.2",
+        "doctrine/doctrine-orm-module": "^2.1.3",
         "laminas-api-tools/api-tools-provider": "^1.2",
-        "laminas/laminas-coding-standard": "~1.0.0",
+        "laminas/laminas-coding-standard": "^2.1.4",
```

Comment:
This should be using the tilde versioning rules; updates to next minor could potentially flag _more_ things than current minor does, which would mean a new PR that gets the next minor version ends up flagging things unrelated to the PR changes.

I can take care of this during merge.